<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarTarea</class>
 <widget class="QDialog" name="AgregarTarea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Tarea</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/iconos/Recursos/lista-de-tareas.png</normaloff>:/iconos/Recursos/lista-de-tareas.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tarea:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inTarea">
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Prioridad:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="cmdAlta">
        <property name="text">
         <string>Alta</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="cmdMedia">
        <property name="text">
         <string>Media</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="cmdBaja">
        <property name="text">
         <string>Baja</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tipo:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="inTipo"/>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="inFinalizada">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Finalizada:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fecha</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QDateEdit" name="inFecha">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2021</year>
          <month>8</month>
          <day>16</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCalendarWidget" name="cmdCalendario"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>275</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdAgregar">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>inFecha</sender>
   <signal>userDateChanged(QDate)</signal>
   <receiver>cmdCalendario</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdCalendario</sender>
   <signal>clicked(QDate)</signal>
   <receiver>inFecha</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
